### PR TITLE
First release

### PR DESCRIPTION
**Pull request by FilippoBau**

### CODE REVIEW

**Review by himazawa — CHANGES_REQUESTED** [review]
Need improvements

File: cmd/aws/aws.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+func HandleRequest(ctx context.Context, event interface{}) (string, error) {
+	fmt.Println("event", event)
```
[the remark this answers is not in the document]
Comment (FilippoBau) [reply cmd/aws/aws.go:11]: The script for the aws part has not been implemented yet, this is the default script

File: cmd/aws/aws.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+	return "Hello world", nil
```
[the remark this answers is not in the document]
Comment (FilippoBau) [reply cmd/aws/aws.go:13]: The script for the aws part has not been implemented yet, this is the default script

File: cmd/azure/azure_test.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+		{"/subscriptions/123/resourceGroup/myResourceGroup/resources/456", "", true},
+		{"/subscriptions/123/resourceGroups/myResourceGroup", "myResourceGroup", false},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.resourceID, func(t *testing.T) {
+			got, err := getResourceGroupFromResourceID(tt.resourceID)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("getResourceGroupFromID()= %v, error = %v, wantErr %v", got, err, tt.wantErr)
+				return
```
Comment (himazawa) [inline cmd/azure/azure_test.go:27]: This is blocking the execution of other tests, why?

**Review by himazawa — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -32,3 +32,8 @@ override.tf.json
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+
+infra/tf_generated_azure
+cmd/parsing/
```
[the remark this answers is not in the document]
Comment (FilippoBau) [reply .gitignore:37]: These are routes used for temporary activities or debugging purposes. They are in gitignore to avoid unintended push.

**Review by himazawa — APPROVED** [review]

**Pull request merged**